src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private final String color;
+    private final double areaCircle;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
+        this.color = color;
+        this.areaCircle = areaCalculator();
+    }
+
+    @Override
+    public double areaCalculator() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String drawable() {
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/1dd48bf3-56f7-40f6-a076-6b8b0a4686bd)

```suggestion
    public void draw() {
```